src/state/types.ts

```diff
@@ -0,0 +1,14 @@
+export type KindeUser = {
+  given_name: string | null;
+  id: string | null;
+  family_name: string | null;
+  email: string | null;
+  updated_at: number;
```

Comment:
`updated_at` isn't actually exposed anywhere in the user object currently, so we can remove this for now